src/api/get-installations.js

```diff
@@ -59,6 +86,7 @@ async function findWindowsBrowsers () {
     if (edgeAlias)
         installations['edge'] = edgeAlias;
 
+
```

Comment:
four - is my mistake, forgot to fix argument.
About other cases: i thought, it's more safe, because at moment '//do smthg'  there can be any other call to process.stdout.write()